src/value-indexer/src/test/scala/VerifyValueIndexer.scala

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
instead of adding nulls to the levels, which didn't work out in a lot of ways, I added a special attribute to the metadata to specifying whether null levels exist